jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/actions/CaseActionsViewImpl.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.workbench.cm.client.actions;
+
+import java.util.List;
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+
+import org.jboss.errai.common.client.dom.Div;
+import org.jboss.errai.common.client.dom.Event;
+import org.jboss.errai.common.client.dom.HTMLElement;
+import org.jboss.errai.ui.client.local.spi.TranslationService;
+import org.jboss.errai.ui.shared.api.annotations.DataField;
+import org.jboss.errai.ui.shared.api.annotations.EventHandler;
+import org.jboss.errai.ui.shared.api.annotations.ForEvent;
+import org.jboss.errai.ui.shared.api.annotations.Templated;
+import org.jbpm.workbench.cm.client.util.AbstractView;
+import org.jbpm.workbench.cm.model.CaseActionSummary;
+import org.uberfire.mvp.Command;
+
+@Dependent
+@Templated
+public class CaseActionsViewImpl extends AbstractView<CaseActionsPresenter> implements CaseActionsPresenter.CaseActionsView {
+
+    public static final String AVAILABLE_ACTIONS = "AVAILABLE";
+    public static final String INPROGRESS_ACTIONS = "IN_PROGRESS";
+    public static final String COMPLETED_ACTIONS = "COMPLETED";
+    @Inject
+    @DataField("available-actions")
+    CaseSimpleActionListViewImpl availableActions;
+    @Inject
+    @DataField("inprogress-actions")
+    CaseSimpleActionListViewImpl inprogressActions;
+    @Inject
+    @DataField("completed-actions")
+    CaseSimpleActionListViewImpl completedActions;
+    @Inject
+    @DataField("actions")
+    private Div actionsContainer;
+    private Command taskAddCommand;
+
+    @Inject
+    private TranslationService translationService;
+
+    @Override
+    public void init(final CaseActionsPresenter presenter) {
+        this.presenter = presenter;
+        availableActions.init(presenter);
+        inprogressActions.init(presenter);
+        completedActions.init(presenter);
+    }
+
+    @Override
+    public void setActionAddCommand(Command taskAddCommand) {
```

Comment:
How about renaming this method to setTaskAddCommand?
The reason is that "setAction" part in the method name implies a more general method which can be applied to any case action which can be added dynamically (like Sub-process for example).
Furthermore, I see also methods on the server side like addDynamicTask, so taking that into account I would go even further and rename this method to setUserTaskAddCommand and the Command instance to userTaskAddCommand.

The same naming ambiguity I see in CaseActionsPresenter, which has a method addDynamicAction that actually only handles dynamic user task.
At this point, this naming might be irrelevant but as code base grows with new dynamic actions being added on UI side, it may make code hard-to-read and understand.